app/src/main/java/org/simple/clinic/recentpatientsview/RecentPatientsViewController.kt

```diff
@@ -53,14 +55,17 @@ class RecentPatientsViewController @Inject constructor(
     val recentPatientsStream = Observables
         .combineLatest(currentFacilityStream, patientConfig)
         .switchMap { (facility, config) ->
-          patientRepository.recentPatients(facility.uuid, limit = config.recentPatientLimit)
+          patientRepository.recentPatients(facility.uuid, limit = config.recentPatientLimit + 1)
```

Comment:
Can you add a comment here about why you are doing this?

Reply:
Added comment